docs/docs/actions.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: "Actions"
+disableTableOfContents: true
+jsdoc: "/(gatsby[/]src[/]redux[/]actions[/]public.js)|(gatsby[/]src[/]redux[/]actions[/]restricted.js)$/"
```

Comment:
this works?

- combine 2 filenames
- escape `.`
```suggestion
jsdoc: "/(gatsby[/]src[/]redux[/]actions[/](public|restricted)[.]js)$/"
```

Reply:
Much more elegant.